cfgov/jinja2/v1/govdelivery-subscribe/error/index.html

```diff
@@ -20,7 +20,6 @@
         <li>Check back often to follow our progress and find new sources of information.</li>
         <li><a href="/contact-us/">Let us know</a> if there’s a particular line of work you want to hear about regularly.</li>
     </ol>
-    <!-- TODO: Commented out until email signup is ready to use.
```

Comment:
This page should be showing two consecutive error explanation blocks? It's got a `That didn’t work.` H1 and bullets followed by a `Please try again.` H1 and bullets. Seems odd.